GOVERNANCE.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Knative Governance Framework"
+linkTitle: "Governance"
+weight: 20
+type: "docs"
+aliases:
+- /contributing/governance/
+---
+
+Knative Governance is broken into three components. At a high level, the Knative
+Steering Committee (KSC) is the "root" level governance body, but the Knative
+Trademark Committee (KTC) operates within a protected governance umbrella in
+parallel with the KSC. As such, the KSC may enforce Code of Conduct sanctions or
```

Comment:
s/enforce/investigate and enforce/
s/sanctions or the like against/actions involving/